tests/runner.js

```diff
@@ -173,7 +189,13 @@ function printFile(filename) {
 
   args.push(filename);
 
-  return cp.spawnSync(parser, args).stdout.toString("utf8");
+  let result = cp.spawnSync(parser, args);
+  
+  return {
+    result: result.stdout.toString("utf8"),
+    status: result.status,
+    errorOutput: result.stderr
+  }
```

Comment:
I'm not sure about this. The idea was to give an informative error when a test file `react.res` is failing to parse, because otherwise tests would pass and an empty string will be stored as snapshot.

But maybe it's not necessary.